app/views/partials/theme/simple/items/show.liquid

```diff
@@ -88,3 +110,53 @@
     </div>
   </div>
 </div>
+
+
+<div class="container">
+  <p class="text-2xl text-gray-900 ">Reviews</p>
+  <div class="recent-reviews">
+    {% for review in item.reviews %}
+      <div class="relative py-2 border-t border-blue-300">
+        <div class="flex flex-wrap mb-4">
+          <div class="rating">
+            <div class="rating-point" style="width: {{ review.rating | divided_by:5 | times:100 }}%"></div>
+          </div>
+          <span class="rating-score">{{ review.rating | round: 2 }}</span>
+        </div>
+        <p><b>{{ review.owner}}</b> : {{ review.comment }}</p>
+      </div>
+    {% endfor %}
+  </div>
+  
+  {%- if context.current_user.id %}
+    <form action="/reviews/create" method="post" class="form-review border border-blue-300 rounded" >
+      <p class="text-center text-2xl text-gray-900 ">Add New Review</p>    
+      {% include 'authenticity_token' %}
+      <input type="hidden" name="item[id]" value="{{ item.id }}">
+      <input type="hidden" name="item[uuid]" value="{{ item.uuid }}">
+      <input type="hidden" name="item[owner]" value="{{ context.current_user.email }}">
```

Comment:
you will have this parameter on the server-side, no need to send it as a hidden argument